src/components/SplitButton/SplitButton.jsx

```diff
@@ -182,6 +182,9 @@ const SplitButton = createClass({
 							hasOnlyIcon
 							isActive={isExpanded}
 							kind={kind}
+							isDisabled={_.every([primaryButtonProps, ...buttonChildProps], {
```

Comment:
/nit You can shorten this a little by using the property iterator shorthand:
`isDisabled={_.every([primaryButtonProps, ...buttonChildProps], 'isDisabled')}`